### PR TITLE
[Feature Fix][Staging] Refactor css file for ace editor 

### PR DESCRIPTION
**Pull request by haoyuchen1992**
### Purpose
When loading wiki page, some css files are rendered very slowly. It is because the css file is putted at the end of body. This PR is mainly to resolve [this bug on Trello](https://trello.com/c/28uDIpje/78-wiki-page-css-flashing). It refactors the css code for wiki editor.

Beside, it changes button text for one modal.
 
### Changes
1) Add `file-view-editor.css` for editing docs tab
2) Put css file inside `stylesheets()`
3)Based on style guideline,  change the button text (submit -> save, submit-> apply)

### Side Effect
None.

### CODE REVIEW

File: website/static/css/file-view-editor.css [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/* Style for Ace wiki editor */
+.wiki-editor {
```
Comment (sloria) [inline website/static/css/file-view-editor.css]: Would it make sense to add these rules to `file-view-page.css`? That would save an extra request for a stylesheet.

Comment (haoyuchen1992) [inline website/static/css/file-view-editor.css]: That makes sense! I will change it and add proper comments.

File: website/templates/profile/account.mako [hunk 2]
```diff
@@ -3,6 +3,10 @@
 
 <%def name="title()">Account Settings</%def>
 
+<%def name="stylesheets()">
+   <link rel="stylesheet" href='/static/css/pages/account-setting-page.css'>;
```
Comment (sloria) [inline website/templates/profile/account.mako]: To be safe, best to call `${parent.stylesheets()}` here.

File: website/templates/profile/addons.mako [hunk 3]
```diff
@@ -1,5 +1,11 @@
 <%inherit file="base.mako"/>
 <%def name="title()">Configure Add-ons</%def>
+
+<%def name="stylesheets()">
```
Comment (sloria) [inline website/templates/profile/addons.mako]: Same here; call the parent method.

File: website/templates/profile/notifications.mako [hunk 4]
```diff
@@ -1,5 +1,10 @@
 <%inherit file="base.mako"/>
 <%def name="title()">Notifications</%def>
+
+<%def name="stylesheets()">
+   <link rel="stylesheet" href='/static/css/pages/account-setting-page.css'>;
```
Comment (sloria) [inline website/templates/profile/notifications.mako]: Here as well; call the parent method.

**Comment by sloria** [comment]
Just some minor fixes and this is good to merge.

**Pull request merged**